Cinema system/Cinema system/Controllers/CinemasController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using CinemaSystem.Models;
+
+namespace Cinema_system.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CinemasController : ControllerBase
+    {
+
+        CinemaDBContext db;
+
+        public CinemasController(CinemaDBContext context)
+        {
+            db = context;
+        }
+
+        [HttpGet]
+        public IList<Cinema> Get()
+        {
+            return db.Cinemas.ToList();
+        }
+
+        [HttpGet("{id}")]
+        public Cinema Get(int id)
+        {
+            return db.Cinemas.Find(id);
+        }
+
+        [HttpPost]
+        public void Post(Cinema cinema)
+        {
+            db.Cinemas.Add(cinema);
+            db.SaveChanges();
+        }
+
+        [HttpPut("{id}")]
+        public void Put(Cinema cinema)
+        {
+            db.Cinemas.Update(cinema);
+            db.SaveChanges();
+        }
+
+        [HttpDelete("{id}")]
+        public void Delete(int id)
+        {
```

Comment:
Why void return type? Shouldn't we return something if we want to create REST API?